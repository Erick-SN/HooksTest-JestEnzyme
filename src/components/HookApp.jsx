import React from 'react';

const HookApp = () => {
  return (
    <>
      <h1>From HookApp</h1>
    </>
  );
};

export default HookApp;
