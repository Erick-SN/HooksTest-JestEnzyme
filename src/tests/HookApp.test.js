import React from 'react';
import { shallow } from 'enzyme';
import HookApp from '../components/HookApp';
describe('HookApp Test', () => {
  test('should render HookApp component', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
