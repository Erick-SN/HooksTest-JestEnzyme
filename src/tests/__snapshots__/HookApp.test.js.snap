// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp Test should render HookApp component 1`] = `
<Fragment>
  <h1>
    From HookApp
  </h1>
</Fragment>
`;
